x-pack/plugins/observability_solution/observability_onboarding/public/application/observability_onboarding_flow.tsx

```diff
@@ -60,6 +61,10 @@ export function ObservabilityOnboardingFlow() {
               <BackButton />
               <CustomLogsPanel />
             </Route>
+            <Route path="/otelLogs">
```

Comment:
nit: I appreciate the old Quickstart flows used camelCasing for URLs but Kibana and ES use underscores/hyphens to separate words in URLs so would be good to follow that for consistency